installation/docker.md

```diff
@@ -93,12 +93,13 @@ docker run \
         -v /opt/openhab/userdata:/openhab/userdata \
         -v /opt/openhab/addons:/openhab/addons\
         -d \
-        --user=<uid> \
+        -e USER_ID=<uid> \
+        -e GROUP_ID=<gid> \
         --restart=always \
         openhab/openhab:<version>-<architecture>-<distributions>
 ```
 
-Where `<uid>` is the user ID number for the `openhab` user which you can obtain using the command `id openhab`, `<version>` is the version of openHAB, `<architecture>` is the architecture of your system and `<distributions>` is the base system (debian or alpine).
+Where `<uid>` is the user ID number for the `openhab` user which you can obtain using the command `id openhab`, `<gid>` is the group ID number for the `openhab` user, `<version>` is the version of openHAB, `<architecture>` is the architecture of your system and `<distributions>` is the base system (debian or alpine).
```

Comment:
This long sentence might be difficult for a non-English speaking person to understand.  Consider either a list, or four separate sentences.

Reply:
Totally right.
Will change it.

Reply:
That's exactly what I mean.
I will rebuild the documentation soon anyway.
Technically, the change is OK.

Reply:
Ah ok.
Sorry then I misunderstood that.

Reply:
Way better now:
![image](https://user-images.githubusercontent.com/7161177/32790520-f3d6ea2a-c95e-11e7-8190-55b2f46f77ab.png)